catalog/istio-stack/README.md

```diff
@@ -0,0 +1,154 @@
+# istio-stack
+
+This stack made it easier to setup a service mesh with [istio](https://istio.io/latest/).  
+It provides the setup for [istio-operator](https://github.com/stevehipwell/helm-charts/tree/master/charts/istio-operator) with sensible defaults
```

Comment:
sorry, didnt see it before. I think this link is not the correct anymore?

Reply:
adjust this sentence